### PR TITLE
Should be fix crashs on Android by make the Gamelistactivity hard to kill

### PR DESCRIPTION
**Pull request by Mullin**

### CODE REVIEW

**Comment by lioncash** [comment]
@dolphin-emu-bot rebuild

File: Source/Android/src/org/dolphinemu/dolphinemu/gamelist/GameListFragment.java [hunk 1]
```diff
@@ -128,6 +128,7 @@ public void onListItemClick(ListView listView, View view, int position, long id)
 		Intent intent = new Intent(getActivity(), EmulationActivity.class);
 		intent.putExtra("SelectedGame", item.getPath());
 		startActivity(intent);
+		startForeground();
```
Comment (lioncash) [inline Source/Android/src/org/dolphinemu/dolphinemu/gamelist/GameListFragment.java]: This is incorrect entirely. startForeground isn't part of the ListFragment or the Fragment class it inherits from.

File: Source/Android/src/org/dolphinemu/dolphinemu/gamelist/GameListFragment.java [hunk 2]
```diff
@@ -123,11 +124,29 @@ public void onListItemClick(ListView listView, View view, int position, long id)
...
+		note.setLatestEventInfo(this, "Dolphin Emulator",
+                              "Running",
+                              pi);
+		note.flags|=Notification.FLAG_NO_CLEAR;
 
 		// Start the emulation activity and send the path of the clicked ROM to it.
 		Intent intent = new Intent(getActivity(), EmulationActivity.class);
 		intent.putExtra("SelectedGame", item.getPath());
 		startActivity(intent);
+		startForeground(1337, note);
```
Comment (lioncash) [inline Source/Android/src/org/dolphinemu/dolphinemu/gamelist/GameListFragment.java]: As I said before: startForeground does ->not <- exist as a part of the fragment or ListFragment or Fragment class.

Comment (Mullin) [inline Source/Android/src/org/dolphinemu/dolphinemu/gamelist/GameListFragment.java]: Due to lack of memory GamelistActivity is killed. I'm trying to make a GamelistActivity Service for delay / prevent killing by LMK.

Comment (lioncash) [inline Source/Android/src/org/dolphinemu/dolphinemu/gamelist/GameListFragment.java]: 'Tricking' Android into thinking an Activity that isn't in the foreground is in the foreground is bad design. Either way if Android wants to kill that Activity, it'll do it regardless (even if implemented as a Service).

A Service should only be used when an application needs to perform a long-running operation in the background, or to enable a functionality in other applications. The game list does neither of these.

>Sorry :(
No need to be sorry,

**Comment by Stevoisiak** [comment]
@dolphin-emu-bot rebuild

**Pull request closed without merging**